### DISCUSSION TITLE
Is there a way to assign weight to particular repos for "Most Used Languages"?

### DISCUSSION THREAD

**Discussion by isayakmondal** [anuraghazra/github-readme-stats#2692] [Q&A] [answered]
So one of my repo has a very large `.ipynb file`, which is messing with my language card statistics, so ideally, I want to give less weight to that repo if possible. I read through the docs and find out that it was possible globally, but I was wondering if it's possible for a particular repository. 
Thank you!

**Comment by qwerty541** [comment, marked as answer]
This repository does not provide ability to assign custom weight to particular repositories but you can exclude some repositories from your language card stats using `exclude_repo` option. See: https://github.com/anuraghazra/github-readme-stats#language-card-exclusive-options

If you want exactly assign custom weight you can submit an issue with feature request or implement it yourself and open pull request.

**Reply by rickstaa** [reply]
Thanks for the answer! I think we will not be implementing this, since the `exclude_repo` option together with https://github.com/anuraghazra/github-readme-stats/pull/2409 will already provide users with a way to adjust the card to their liking 🤔.
